packages/login-ui/components/GenerateWallet/lang.js

```diff
@@ -4,10 +4,10 @@
  */
 
 export const en = {
-  enterPassword: 'Enter password for the new wallet:',
-  password: 'Password',
-  passwordWarning: 'At least 8 characters',
-  iUnderstand: 'I\u00a0understand',
-  continue: 'Download Wallet',
-  walletSuccess: 'Your wallet has been generated',
+  title: 'Download a Wallet File',
+  description: 'You can use this wallet file in password recovery option to\n' +
+  '              make your account available in another browser, for example.',
+  descriptionExtra: 'The file is protected by the same password as your created before.',
```

Comment:
Typo: The file is protected by the same password as you[r] created before

your -> you